pandas/io/tests/parser/common.py

```diff
@@ -1292,3 +1292,27 @@ def test_read_duplicate_names(self):
         expected = DataFrame([[0, 1, 2], [3, 4, 5]],
                              columns=['a', 'b', 'a.1'])
         tm.assert_frame_equal(df, expected)
+
+    def test_inf_parsing(self):
```

Comment:
No need to change anything for Python engine.  `floatify` fix fixes that issue :smile: